### ISSUE TITLE
Can you support Top Divider?

### ISSUE THREAD

**Issue by Airsaid** [Fondesa/RecyclerViewDivider#22]
Don't know if the author has plans to support it?

**Comment by Fondesa** [comment]
Related to https://github.com/Fondesa/RecyclerViewDivider/issues/21 
Currently, I have no plans to support it.

**Comment by fondesa** [comment]
@Airsaid Since other users asked it, I'll add this feature in the milestone [4.x](https://github.com/fondesa/recycler-view-divider/milestone/2).
The updates will be done on the issue #68.

**Comment by fondesa** [comment]
Just released the version [3.2.0](https://github.com/fondesa/recycler-view-divider/releases/tag/3.2.0) which contains `showFirstDivider` to show the top divider.

**e.g.** (kotlin)

```kotlin
context.dividerBuilder()
    .showFirstDivider()
    .build()
    .addTo(recyclerView)
```
